pages/[resourceType]/create.tsx

```diff
@@ -49,6 +49,77 @@ const CreateResourcePage = () => {
       </Center>
     </div>
   );
+
+  //handles what will happen when the submit button is clicked.
+  async function editorSubmitHandler() {
+    let customMessage: NotificationProps["message"] = <div>Problem connecting to server</div>;
+    let notifProps: NotificationProps = {
+      message: customMessage,
+      color: "red",
+      icon: <X size={18} />,
+      autoClose: false,
+    };
+
+    await fetch(`${process.env.NEXT_PUBLIC_DEQM_SERVER}/${resourceType}`, {
```

Comment:
I'm assuming this `await` is needed to make sure the notifications show after the promise either resolves or rejects. There is actually built in functionality with this. Take a look at [Promise.finally](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/finally)

I think putting the clean/show notification code in a `.finally` block gets rid of the need for this await? Then this whole function doesn't need to be async at all right?